controllers/alerts/reconciler.go

```diff
@@ -37,9 +38,23 @@ type MonitoringReconciler struct {
 }
 
 func NewMonitoringReconciler(ci hcoutil.ClusterInfo, cl client.Client, ee hcoutil.EventEmitter, scheme *runtime.Scheme) *MonitoringReconciler {
-	deployment := ci.GetDeployment()
-	namespace := deployment.Namespace
-	owner := getDeploymentReference(deployment)
+	var namespace string
+	var owner metav1.OwnerReference
+	if ci.IsRunningLocally() {
+		owner = metav1.OwnerReference{
+			APIVersion:         appsv1.SchemeGroupVersion.String(),
+			Kind:               "Deployment",
+			Name:               "local",
+			UID:                uuid.NewUUID(),
+			BlockOwnerDeletion: pointer.Bool(false),
+			Controller:         pointer.Bool(false),
+		}
+		namespace = "kubevirt-hyperconverged"
```

Comment:
Can we get the actual NS somehow? Not sure it will be "kubevirt-hyperconverged" in all cases.

Reply:
done